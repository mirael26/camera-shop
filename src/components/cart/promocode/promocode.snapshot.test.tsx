import { renderWithReduxAndRouter } from '../../../test/helpers/render-with-redux-and-router';
import Promocode from './promocode';

test('Promocode component renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<Promocode/>);
  expect(asFragment()).toMatchSnapshot();
});
