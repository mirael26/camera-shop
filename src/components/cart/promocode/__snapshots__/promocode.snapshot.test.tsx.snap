// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Promocode component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="basket__promo"
  >
    <p
      class="title title--h4"
    >
      Если у вас есть промокод на скидку, примените его в этом поле
    </p>
    <div
      class="basket-form"
    >
      <form
        action="#"
      >
        <div
          class="custom-input"
          data-testid="promocode-container"
        >
          <label>
            <span
              class="custom-input__label"
            >
              Промокод
            </span>
            <input
              name="promo"
              placeholder="Введите промокод"
              type="text"
              value=""
            />
          </label>
        </div>
        <button
          class="btn"
          type="submit"
        >
          Применить
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
